pkg/deploy/graph/analysis/dc_test.go

```diff
@@ -0,0 +1,76 @@
+package analysis
+
+import (
+	"path/filepath"
+	"testing"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/meta"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/resource"
+
+	osgraph "github.com/openshift/origin/pkg/api/graph"
+	osgraphtest "github.com/openshift/origin/pkg/api/graph/test"
+	kubeedges "github.com/openshift/origin/pkg/api/kubegraph"
+	"github.com/openshift/origin/pkg/api/latest"
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+	deploygraph "github.com/openshift/origin/pkg/deploy/graph/nodes"
+)
+
+func TestCheckMountedSecrets(t *testing.T) {
```

Comment:
Looks like more than 50% of this test is loading a YAML test fixture using stuff you're not interested in testing here (e.g. resource builder)... what's the advantage over a code based test fixture?